picky/src/key/ec.rs

```diff
@@ -4,7 +4,7 @@ use picky_asn1_x509::{oids, private_key_info};
 
 pub(crate) struct EcdsaKeypair<'a> {
     pub(crate) private_key: Vec<u8>,
-    pub(crate) public_key: &'a [u8],
+    pub(crate) _public_key: &'a [u8],
```

Comment:
I think we can remove this one

Reply:
This field is now used but only used in test `sign_and_verify_compatibility_with_ring`

Reply:
If I remember correctly, the new crypto libraries you're using allow to obtain the public key starting from the private key. I would remove the `_public_key` attribute entirely and change the unit test to obtain the public key starting from the private one.

I hope I explained myself properly :sweat_smile:

Reply:
Yea, I understood your suggestion, very clearly! I once thought that the `public key` should be read using `crate::key::ec::EcdsaKeypair::try_from` because it is provided in this crate. Now I changed the way to read the PEM-encoded key using `p256`/`p384` APIs as your suggestions. In this way the API for `crate::key::ec::EcdsaKeypair::try_from` will not be tested. Hope that it is not an important problem

Reply:
I like the change.